docs/reference-guides/block-api/block-transforms.md

```diff
@@ -281,7 +281,7 @@ to run them through any block transforms.
 
 ### Shortcode
 
-This type of transformations support the _from_ direction, allowing blocks to be created from shortcodes. It's applied as part of the `raw` transformation process.
+This type of transformations support the _from_ direction, allowing blocks to be created from shortcodes. It's applied as part of the `raw` transformation process. This means that the conversion happens when running "Convert to blocks" on the classic block. To transform the shortcode block use the `block` transformation process. 
```

Comment:
The raw transformation process is also used during copy-paste operations e.g: if a shortcode is pasted in the editor it is transformed in the equivalent block. I guess we could say "This means that the conversion happens when running "Convert to blocks" on the classic block or when pasting a shortcode on the block editor..."